packages/styling/src/interfaces/ITheme.ts

```diff
@@ -16,6 +17,7 @@ export interface ITheme {
    * Customizer will not get the global styles applied to them.
    */
   disableGlobalClassNames: boolean;
+  typography: ITypography;
```

Comment:
We need to somehow make sense of this vs the 'fonts' property in this same interface. Maybe we should roll the stuff in here into IFontStyles instead? Please chat with David to figure that out (or whoever the owner of font stuff is).

Reply:
One important note is that anything we check into here is PERMANENT, in order to maintain backwards compatibility. Basically, imagine someone made a theme today and saved it as a JSON blob, and then never ever updates the theme ever again for the next twenty years. Our interface needs to support that. Thus, if we have two places we're pulling the font from, all existing code has to check two separate places, which is quite a large burden to put on all work going forward.

Reply:
@phkuo 
I'm not sure how to stage this. I'd like to get this checked in in a way that is consumable, but also that we aren't giving a requirement or even documentation or guidance to partners to consume it.

One thing we could do is to mark the prop as experimental and to avoid using until finalized? Can you think of any other ways that we could get this in with the freedom to change and not break people?

Reply:
If we want to go with the experimental comment route, make sure to comment on the intent that this is to replace fontStyles in a future release and that it is undergoing review.

Reply:
As long as we make this optional (aka transparent to partners), and we have no non-experimental cases dependent on it, I think it's ok to add.